### PR TITLE
soft set ulimit in gitlab-ci.yml

### PR DESCRIPTION
**Pull request by SkySkimmer**
Some of the scripts set it to a higher value (eg color)

No idea how it worked before.

### CODE REVIEW

**Review by silene — CHANGES_REQUESTED** [review]
Printing the value of `ulimit -s` just before modifying it seems like noise.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -109,7 +109,8 @@ before_script:
     # flambda can be pretty stack hungry, specially with -O3
     # See also https://github.com/ocaml/ocaml/issues/7842#issuecomment-596863244
     # and https://github.com/coq/coq/pull/11916#issuecomment-609977375
-    - ulimit -s 16384
+    - ulimit -s
+    - ulimit -S -s 16384
```
Comment (JasonGross) [inline .gitlab-ci.yml:113]: Shouldn't we make the hard limit unlimited?

**Comment by SkySkimmer** [comment]
I'll put it in the before_script for if we get curious.

**Review by silene — APPROVED** [review]

**Comment by silene** [comment]
@coqbot merge now

**Pull request merged**